emission/net/auth/token_list.py

```diff
@@ -10,21 +10,13 @@
 import json
 import traceback
 import requests
+import emission.storage.decorations.token_queries as tq
 
 class TokenListMethod(object):
-    def __init__(self):
+    def __init__(self, source='conf/net/auth/token_list.json', userid=0):
 
-        # import emission.storage.decorations.token_queiries as tq
-        # key_file = tq.get_all_tokens()
-
-        key_file = open('conf/net/auth/token_list.json')
-        key_data = json.load(key_file)
-        key_file.close()
-        self.token_list_file = key_data["token_list"]
-        with open(self.token_list_file) as tlf:
-            raw_token_list = tlf.readlines()
-        self.token_list = [t.strip() for t in raw_token_list]
-        raw_token_list = None
+        tq.insert_from_json(source, userid)
+        self.token_list = tq.get_all_tokens()
```

Comment:
@aGuttman this is not what we want.
Recall that the original problem from https://github.com/e-mission/e-mission-docs/issues/790 was that
> Container image has the files baked in, so we would need to rebuild it every time there is a new deployment

Our goal behind storing it in the database is so that the containers can all be identical
If you load the data in the init for `emission/net/auth/`:
(a) it will be loaded every time we try to validate a token
(b) it will attempt to load from the container, which means that the file needs to be on the container, which means that the containers will need to be different anyway

The load needs to happen in a separate script as indicated in
https://github.com/e-mission/e-mission-docs/issues/790#issuecomment-1247238718

The code here should only *read* from the database.

Reply:
Where in the directory structure should this new script live?

Reply:
scripts are, per convention, in `bin`
you should be able to find plenty of examples there that also use `argparse` for high quality command line processing.